tpp-app/src/home/components/BackButtonComponent.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import { View, StyleSheet } from 'react-native';
+import { Button } from 'react-native-elements';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+
```

Comment:
Just curious, but were you thinking that this could be the back button for the entire app?